diesel/src/expression/array_comparison.rs

```diff
@@ -13,7 +13,8 @@ pub struct In<T, U> {
 
 #[derive(Debug, Copy, Clone)]
 pub struct NotIn<T, U> {
-    in_expr: In<T, U>
+    left: T,
```

Comment:
I'm not sure if left is a good name for this but I don't have a better idea right now